suite-native/module-send/src/components/FiatAmountInput.tsx

```diff
@@ -43,9 +43,15 @@ export const FiatAmountInput = ({
         valueTransformer: fiatAmountTransformer,
     });
 
+    // Validation is assigned to the crypto field, so we need to check if that field has an error.
+    const { hasError } = useField({
+        name: cryptoFieldName,
+    });
+
     const handleChangeValue = (newValue: string) => {
```

Comment:
it seems that it does not support use of decimal comma `,`


https://github.com/user-attachments/assets/b0b2d8c3-ff26-44bc-a103-25a71bef254e

Reply:
Works now nicely 💪